model/generate.go

```diff
@@ -727,14 +727,24 @@ type projectMaps struct {
 }
 
 // validateMaxTasksAndVariants validates that the GeneratedProject contains fewer than 100 variants and 1000 tasks.
-func (g *GeneratedProject) validateMaxTasksAndVariants() error {
+func (g *GeneratedProject) validateMaxTasksAndVariants(ctx context.Context, p *Project) error {
 	catcher := grip.NewBasicCatcher()
```

Comment:
nit: since the catcher only starts being used after L736, can we move the definition down there?